src/test/addresses_deployers.sol

```diff
@@ -14,13 +14,13 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity 0.6.12;
+pragma solidity 0.8.16;
 
 contract Deployers {
 
     address[] public addr;
 
-    constructor() public {
+    constructor() {
         addr = [
```

Comment:
I don't recall the rationale but maybe was added to label the addresses in a more effective way.

Reply:
I don't think it is more effective, just more annoying...

Reply:
> I don't recall the rationale but maybe was added to label the addresses in a more effective way.

It was so we could better distinguish current and past skip wards.  The comments weren't being used consistently so this was a way of forcing labels where the addresses needed to be categorized (goerli where some get skipped vs mainnet where none do).

We can look at how to better achieve that in goerli, but I do not believe we should sacrifice utility for (fake) consistency between mainnet and goerli.  to be clear: i say fake because these addresses are used differently between goerli and mainnet and the addresses themselves are different, so I don't see huge benefit to shoehorning the same formatting